ogr_fdw.c

```diff
@@ -1265,3 +1283,286 @@ ogrEndForeignScan(ForeignScanState *node)
 
 
 #endif /* PostgreSQL 9.3 version check */
+
+#if PG_VERSION_NUM >= 90500
+/*
+ * PostgreSQL 9.5 or above.  Import a foreign schema
+ */
+static List *
+ogrImportForeignSchema(ImportForeignSchemaStmt *stmt, Oid serverOid)
+{
+	List	   *commands = NIL;
+	ForeignServer *server;
+	
+	List *options;
+	ListCell *cell;
+	char *sGeomType;
+	bool check_schema = false;
+	bool		launder_column_names = true;
+	bool		launder_table_names = true;	
+	StringInfoData buf;
+	OgrConnection ogr;
+	int i;
+	int j;
+	int k;
+	char layer_name[STR_MAX_LEN];
+	char table_name[STR_MAX_LEN];
+	ListCell   *lc;
+	bool include_item = false;
+	OGRDataSourceH ogr_ds = NULL;
+	OGRSFDriverH ogr_dr = NULL;
+	OGRFeatureDefnH ogr_fd = NULL;
+	OGRLayerH ogr_lyr = NULL;
+
+	/** check table prefix if remote_schema asked for is not ogr_all **/
+	check_schema = !( strcmp(stmt->remote_schema, "ogr_all") == 0 );
```

Comment:
Just tried with this:
```

IMPORT FOREIGN SCHEMA *
    LIMIT TO(dashboard, debt_analysis)
    FROM SERVER svr_budget2015 INTO budget2015;

```

And I get:

```
ERROR:  syntax error at or near "*"
LINE 2: IMPORT FOREIGN SCHEMA *
                              ^

********** Error **********

ERROR: syntax error at or near "*"
SQL state: 42601
Character: 24
```

So the only way to do it would be to quote "*"  like:
```
IMPORT FOREIGN SCHEMA "*"
    LIMIT TO(dashboard, debt_analysis)
    FROM SERVER svr_budget2015 INTO budget2015;
```

 which I think makes it even more confusing, because then people will question why you need to quote "*" here and not elsewhere in SQL.